src/sass/bootstrap-asu.scss

```diff
@@ -5,46 +5,49 @@
  * ======================================================================== */
 
 // Load UDS Design Tokens
-@import "design-tokens/extra-vars";
 @import 'design-tokens/variables';
 
 // default path for image assets (assuming the images will be hosted from the web root)
 // Storybook and themes override this for their particular path needs
-$image-assets-path: '../../img' !default;
+$image-assets-path: '/img' !default;
```

Comment:
Same as with the other "manifest" file, we need to make sure updates get made here, but also to keep the details tweaked for the CMS.

The README.md for the theme will need to be updated about these details.